app/models.py

```diff
@@ -59,3 +61,44 @@ class Tasks(db.Model):
 
     def __repr__(self):
         return '<Task {}>'.format(self.task)
+
+
+# Flask-Security
+
+roles_users = db.Table('roles_users',
+        db.Column('user_id', db.Integer(), db.ForeignKey('user.id')),
+        db.Column('role_id', db.Integer(), db.ForeignKey('role.id')))
+
+
+class Role(db.Model, RoleMixin):
+    id = db.Column(db.Integer(), primary_key=True)
+    name = db.Column(db.String(80), unique=True)
+    description = db.Column(db.String(255))
+
+
+class User(db.Model, UserMixin):
+    id = db.Column(db.Integer, primary_key=True)
+    email = db.Column(db.String(255), unique=True)
+    password = db.Column(db.String(255))
+    active = db.Column(db.Boolean())
+    confirmed_at = db.Column(db.DateTime())
+    roles = db.relationship('Role', secondary=roles_users, backref=db.backref('users', lazy='dynamic'))
```

Comment:
one user has many roles?

Reply:
Isn't that how it is? An `admin` is also a `user` and so on?

Reply:
I am asking. 😛 

I thought you have implemented foreign key for role and not many to many

Reply:
Ohh, acha. Nai nai, it is many to many. See the `user_roles` table